homeassistant/components/webrtc/api/client.py

```diff
@@ -0,0 +1,110 @@
+"""Client library for go2rtc."""
```

Comment:
I think the client, model and utils belongs in a 3rd party library in home-assistant-libs org.

Reply:
That is the plan when everything is ready.
I just opened the PR to test the code on multiple machines.